### PR TITLE
restore-config: support --skip-network option

### PR DESCRIPTION
**Pull request by gsanchietti**
Disable service if bridge does not exist.

The service is installed but then disabled. I do not know if this is the right approach.

NethServer/dev#6099

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-dc-network-reset [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+use esmith::NetworksDB;
+use esmith::ConfigDB;
+
+my $ndb = esmith::NetworksDB->open();
+my $cdb = esmith::ConfigDB->open();
+
+# ARGV[0] is the name of the event
+# ARGV[1] is the first real parameter
+if (defined($ARGV[1]) && $ARGV[1] eq '--skip-network') {
+
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-dc-network-reset]:
```suggestion
if (grep(/^--skip-network$/, @ARGV)) {
```

File: root/etc/e-smith/events/actions/nethserver-dc-network-reset [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+my $ndb = esmith::NetworksDB->open();
+my $cdb = esmith::ConfigDB->open();
+
+if (grep(/^--skip-network$/, @ARGV)) {
+    # Disable nsdc if the bridge doesn't exists
+    my $br = $cdb->get_prop('nsdc', 'bridge');
+    if(!$ndb->get($br)) {
+        $cdb->set_prop('nsdc', 'status', 'disabled');
+        $cdb->set_prop('nsdc', 'bridge', '');
```
Comment (DavidePrincipi) [inline root/etc/e-smith/events/actions/nethserver-dc-network-reset:36]: I'd write in the log what happened

```suggestion
        $cdb->set_prop('nsdc', 'bridge', '');
        warn "[NOTICE] nsdc forced to status disabled due to --skip-network flag\n";
```

**Review by DavidePrincipi — APPROVED** [review]
Approved! Anyway I'd add a little log NOTICE...

File: root/etc/e-smith/events/actions/nethserver-dc-network-reset [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        warn "[NOTICE] nsdc forced to status disabled due to --skip-network flag\n";
```
Comment (gsanchietti) [inline root/etc/e-smith/events/actions/nethserver-dc-network-reset]:
```suggestion
        warn "[WARNING] nsdc forced to status disabled due to --skip-network flag\n";
```

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**